src/main/java/com/conveyal/datatools/editor/controllers/api/EditorController.java

```diff
@@ -129,6 +132,10 @@ private void registerRoutes() {
             put(ROOT_ROUTE + ID_PARAM + "/stop_times", this::updateStopTimesFromPatternStops, json::write);
             delete(ROOT_ROUTE + ID_PARAM + "/trips", this::deleteTripsForPattern, json::write);
         }
+
+        if ("stop".equals(classToLowercase)) {
+            patch(ROOT_ROUTE + "/cascadeDeleteStop", this::cascadeDeleteStop, json::write);
```

Comment:
Could we make the endpoint a `delete` or just a `get` endpoint? `patch` requests have some annoying requirements that make things a bit difficult on the front end

Reply:
@philip-cline I've changed to a delete. Note that the stop id now has to be part of the URL and not a query param:
`/api/editor/secure/stop/STOP_ID_HERE/cascadeDeleteStop?feedId=%s&sessionId=test`